trk/src/Screens/LiveClimbTracker/index.js

```diff
@@ -0,0 +1,233 @@
+import React, { useState, useContext, useEffect } from "react";
+import { View, Text, Button, TextInput, StyleSheet, Alert, TouchableOpacity, ScrollView, SafeAreaView } from "react-native";
+import firestore from '@react-native-firebase/firestore';
+import { AuthContext } from "../../Utils/AuthContext";
+import { useFocusEffect, useNavigation } from '@react-navigation/native';
+import DropDownPicker from "react-native-dropdown-picker";
+import usePopularTimesData from "../LiveClimbTracker_Backend/LiveClimbTrackerLogic";
+import CalendarStrip from 'react-native-calendar-strip';
+import { ActivityIndicator } from 'react-native';
+import { BarChart } from "react-native-chart-kit";
+import { Dimensions } from "react-native";
+import moment from "moment-timezone";
+
+const LiveClimbTracker = () => {
+    const navigation = useNavigation(); 
+    const { currentUser } = useContext(AuthContext);
+
+    const [openGymPicker, setOpenGymPicker] = useState(false);
+    const [openClimbPicker, setOpenClimbPicker] = useState(false);
+    const currentDate = moment().tz('America/New_York');
+    const [activeDate, setActiveDate] = useState(currentDate);
+    
+    const [reloadTrigger, setReloadTrigger] = useState(false);
+    const handleReload = () => {
+        setReloadTrigger(prev => !prev); // Toggle the state to trigger a reload
+    };
+
+    const [selectedGymId, setSelectedGymId] = useState(null);
```

Comment:
The only gym we want users to get info on is Movement LIC. So IMO you can put it as default, and only choosable value

Reply:
for now... ;)

In the future, we'll allow users to select more than one, once we have more gyms. Though, we'll want to restrict users' access to only gyms they subscribe to. Not there yet, but soon™